android/src/com/mozilla/vpn/VPNService.kt

```diff
@@ -133,7 +175,10 @@ class VPNService : android.net.VpnService() {
         if (fileDescriptor != null) {
             Log.v(tag, "Got file Descriptor for VPN - Try to up")
             backend.tunnelUp(tunnel, fileDescriptor, config.toWgUserspaceString())
-            this.startSticky()
+            mNotificationBuilder
+                .setContentTitle("Todo: Connected")
```

Comment:
We need to localize these strings... The best would be that the app sends the localized strings to the service as preference.